### PR TITLE
v0.52.0 - addition of subtitle styles to content header. update of fo…

### PR DESCRIPTION
**Pull request by sonnyprince**
_Content header style updates._

screen shot 2018-07-26 at 16 27 54
screen shot 2018-07-26 at 16 28 01

- Addition of content header subTitle styles
- Update to content header font styles

## UI Review Checks

- [x] This PR has been checked with regard to our brand guidelines

## Browsers Tested

- [x] Chrome
- [x] Mobile (i.e. iPhone/Android - please list device)

### List any other browsers that this PR has been tested in:

- [View the Browser Support Checklist](http://fozzie.just-eat.com/documentation/general/browser-support)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,20 +3,31 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.51.3
+------------------------------
+*July 25, 2018*
```
Comment (ashleynolan) [inline CHANGELOG.md]: Bump 😄

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,20 +3,31 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 
+v0.51.3
```
Comment (ashleynolan) [inline CHANGELOG.md]: This is a minor release – patch releases should only be for tiny changes (essentially one line bug fixes really).

File: src/scss/components/_content-header.scss [hunk 3]
```diff
@@ -7,19 +7,39 @@
...
-    padding-bottom: spacing(x2);
+    padding-bottom: spacing();
 }
 
 .c-contentHeader-title {
     padding: 0;
     display: inline-block;
     margin: 0 spacing(x2) 0 0;
+    font-weight: $font-weight-base;
+    font-family: $font-family-headings;
```
Comment (ashleynolan) [inline src/scss/components/_content-header.scss]: Does this not also use a `h` tag?  Might be good to see example markup, as if it's a title probably should be a header semantically…?

File: CHANGELOG.md (hunk 1 above)
Comment (sonnyprince) [reply CHANGELOG.md]: 😄 I _will_ remember to do this

File: src/scss/components/_content-header.scss (hunk 3 above)
Comment (sonnyprince) [reply src/scss/components/_content-header.scss]: It does, was trying to make more generic but I agree with the header semantics so i've removed the font family :)

File: package.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "name": "@justeat/fozzie",
   "title": "Fozzie – Just Eat UI Web Framework",
   "description": "UI Web Framework for the Just Eat Global Platform",
-  "version": "0.51.2",
+  "version": "0.51.3",
```
Comment (DamianMullins) [inline package.json]: You've got v0.52.0 in the changelog and v0.51.3 here.

Comment (ashleynolan) [reply package.json]: 😱 lol

Comment (ashleynolan) [reply package.json]: We should probably write a Danger.js check to ensure these match…

**Pull request merged**